### PR TITLE
remove quo lib

### PR DESCRIPTION
**Pull request by flexsurfer**
https://github.com/status-im/status-mobile/assets/11790366/828d49d6-6a96-4320-8153-6487308b545f

### CODE REVIEW

**Comment by J-Son89** [comment]
thanks for the work @flexsurfer! curious did you actually delete everything or some of it is moved into status-im ns or something?

**Review by J-Son89 — APPROVED** [review]
Looks great @flexsurfer! This will be much clearer for those contributing to the project now. Next stop we can rename quo2 -> quo 😄 🍾

**Review by Parveshdhull — APPROVED** [review]

**Review by siddarthkay — APPROVED** [review]
I trust your nuking!
GG

**Comment by flexsurfer** [comment]
> thanks for the work @flexsurfer! curious did you actually delete everything or some of it is moved into status-im ns or something?

yeah i had to move a few components to status-im, but i simplified them anyway, so they just pure react components

**Review by ilmotta — APPROVED** [review]
@flexsurfer plays a serious game, my gosh!

File: src/quo/animated.cljs
Comment (siddarthkay) [inline src/quo/animated.cljs:1]: Pls DM me where to deliver the cheese naan

**Review by FFFra — APPROVED** [review]

**Comment by FFFra** [comment]
image

**Comment by FFFra** [comment]
When I grow up I wanna be like you

**Review by yqrashawn — APPROVED** [review]

**Comment by churik** [comment]
### ISSUE 1: buttons on `Report a bug` screen are not visible

Shake device > Report a bug

https://github.com/status-im/status-mobile/assets/4557972/04caa4b2-d8ca-460d-b974-803c8e00efeb

### ISSUE 2: all toggles and checkboxes look enabled

Video in comparison with current nightly

https://github.com/status-im/status-mobile/assets/4557972/c058c902-53e3-43a9-afa8-1fce7df8361a

**Comment by flexsurfer** [comment]
thank @churik fixed

**Comment by churik** [comment]
Let's gooooo!
rollercoaster-scared

**Pull request merged**